addons/website_livechat/views/website_livechat.xml

```diff
@@ -2,13 +2,157 @@
 <openerp>
     <data>
 
-<template id="header" inherit_id="website.layout" name="LiveChat Import (internal) Scripts">
-    <xpath expr="//body" position="inside">
-        <t t-if="website and website.channel_id">
-            <t t-raw="website.channel_id.script_internal"/>
-        </t>
-    </xpath>
-</template>
+        <template id="header" inherit_id="website.layout" name="LiveChat Import (internal) Scripts">
```

Comment:
Usually we unindent the templates, because otherwise you are loosing a lot of spaces. You could have
<openerp>
....<data>
<template1/>
<template2/>
....</data>
</openerp>